.travis.yml

```diff
@@ -23,12 +23,16 @@ matrix:
     - php: 7.1
       env:
         - PHPUNIT_TEST=framework
+        # Test with lowest composer requirements (i.e. framework 4.0)
+        - COMPOSER_ARG="--prefer-lowest"
```

Comment:
just ned a RECIPE_VERSION here too, I suppose

Reply:
My hope is that it will pull in its require and require-dev dependencies, it will be enough, but you might be right.

Reply:
You were right. Testing the postgresql suite without a recipe is fine but testing framework, needs at a minimum framework's require-dev dependencies, which the recipe provides.